src/test/users.test.ts

```diff
@@ -0,0 +1,59 @@
+//import { expect } from 'chai';
+import { AppDataSource } from '../data-source';
+import { User } from '../entity/User';
+import { queryGetAllUsers } from './query-users';
+import { expect } from 'chai';
+import { newUser } from '../utils/data-generate-users';
+import { generateFakeToken } from './functions';
+
+describe('query users', async () => {
+  const users = [];
+  const token: string = await generateFakeToken();
+
+  beforeEach(async () => {
+    for (let i = 0; i < 20; i++) {
+      users.push(await newUser());
+    }
+
+    AppDataSource.getRepository(User).save(users);
+  });
+
+  afterEach(async () => {
+    AppDataSource.getRepository(User).delete({});
+  });
+
+  it('should return a vector of users', async () => {
+    const formatArray = users
```

Comment:
```suggestion
    const expectedUsers = users
```

É legal sempre tentar deixar bem claro a variável pelo seu nome e evitar nomes genéricos. Essa variável tem um nome que poderia ser usado pra muitas coisas, concorda?